packages/react-text/src/components/Display/Display.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import { Text } from '../Text/index';
+import { typographyStyles } from '../../typographyStyles/index';
+import { DisplayProps } from './Display.types';
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: typographyStyles.display,
+});
+
+/**
+ * Text wrapper component for the Display typography variant
+ */
+export const Display = React.forwardRef<HTMLElement, DisplayProps>((props, ref) => {
+  const styles = useStyles();
+  const className = mergeClasses(styles.root, props.className);
+
+  return <Text {...props} ref={ref} className={className} />;
+});
```

Comment:
would be nice to have this component following other components pattern of declaration, `useState`, `useStyle` and `render` function spearate

Reply:
This should composition via hooks as we want to avoid additional components in React's tree. Current implementation will output:

```tsx
<Display>
  <Text>
    <span />
  </Text>
</Display>
```

A suggestion below:

```tsx
<Display>
  <span />
</Display>
```

```suggestion
export const Display = React.forwardRef<HTMLElement, DisplayProps>((props, ref) => {
  const state = useText(props, ref);

  useTextStyles(state);
  state.className = mergeClasses(state.className, styles.root, props.className);

  return renderText(state);
});
```

Reply:
Thanks for the suggestion! Applied changes